fesvr/htif.h

```diff
@@ -74,7 +75,7 @@ class htif_t : public chunked_memif_t
   void parse_arguments(int argc, char ** argv);
   void register_devices();
   void usage(const char * program_name);
-
+  unsigned int max_xlen = 0;
```

Comment:
I wonder if HTIF should really describe this as the "maximum" xlen? Here in HTIF, it defines the _only_ xlen allowed in the loaded ELF. Should we call this `expected_xlen` or `enforce_xlen` or perhaps simply `xlen`?

(Also, I considered asking for `std::optional<unsigned>` here, instead of treating 0 as a special value, but in this case it's obvious that 0 is not a valid xlen, so I think the current system is acceptable.)

Reply:
Fair point. I also think expected_xlen is more fitting.